prow/test/integration/test/gerrit_test.go

```diff
@@ -34,11 +50,67 @@ func TestGerrit(t *testing.T) {
 		t.Fatalf("Failed creating gerritClient: %v", err)
 	}
 
+	change := gerrit.ChangeInfo{
+		CurrentRevision: "1",
+		ID:              "1",
+		ChangeID:        "1",
+		Project:         "test-infra",
+		Branch:          "master",
+		Status:          "NEW",
+		Updated:         makeStamp(timeNow),
+		Revisions: map[string]client.RevisionInfo{
+			"1": {
+				Number:  1,
+				Created: makeStamp(timeNow.Add(-time.Hour)),
+			},
+		},
+		Messages: []gerrit.ChangeMessageInfo{
+			{
+				Message:        "Hello",
+				RevisionNumber: 1,
+				Date:           makeStamp(timeNow),
+			},
+		},
+	}
+
+	err = addChangeToServer(change)
+	if err != nil {
+		t.Fatalf("Failed to add change to server: %s", err)
+	}
+
 	resp, err := gerritClient.GetChange(gerritServer, "1")
 	if err != nil {
 		t.Fatalf("Failed getting gerrit change: %v", err)
 	}
 	if resp.ChangeID != "1" {
 		t.Errorf("Did not return expected ChangeID. Want: %q, got: %q", "1", resp.ChangeID)
 	}
+
+	changes := gerritClient.QueryChanges(lastSyncState, 10)
+	if len(changes[gerritServer]) != 1 {
+		t.Errorf("Did not return expected ChangeID. Want: %q, got: %v", "1", len(changes[gerritServer]))
+	}
+
+	reset()
```

Comment:
Remove?

Reply:
No. reset() calls the API on the fake gerrit server to reset the changes in the server. Without this the tests would pass the first run and fail afterward.

Reply:
No. reset() calls the API on the fake gerrit server to reset the changes in the server. Without this the tests would pass the first run and fail afterward.

Reply:
Can you add this as a comment?

Reply:
Done